_posts/2021-09-30-building-opensearch-1.1-distributions.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "Building OpenSearch 1.1 Distributions using Automation"
+authors: 
+  - dblock
+date: 2021-09-30
```

Comment:
Next time: Include the time and TZ offset. I had to debug in production ;)

Reply:
Might be worthwhile to create an issue to automately check for these issues